From fcce14e239ab540acebed85edfb96caac468ccc3 Mon Sep 17 00:00:00 2001
From: Marc Leeman <marc.leeman@gmail.com>
Date: Fri, 6 Dec 2024 10:29:19 +0100
Subject: [PATCH 1/5] build: Fix incorrect OPENVPN_USERNAME in D-Bus autostart
 files

Remove hard coded `openvpn` user in meson files.  The build system
allows to replace the default `openvpn` user with some other name.
This works for most services, but during refactoring, 3 hard coded
references were omitted.

This popped up while packaging for Debian since the system users should
 be prepended with a '_'.

Signed-off-by: Marc Leeman <marc.leeman@gmail.com>
Approved-by: David Sommerseth <davids@openvpn.net>
---
 src/configmgr/meson.build  | 1 -
 src/log/meson.build        | 1 -
 src/sessionmgr/meson.build | 1 -
 3 files changed, 3 deletions(-)

diff --git a/src/configmgr/meson.build b/src/configmgr/meson.build
index 5b452bd..3a00e69 100644
--- a/src/configmgr/meson.build
+++ b/src/configmgr/meson.build
@@ -45,7 +45,6 @@ configure_file(
             'BUSNAME': 'net.openvpn.v3.configuration',
             'SERVICE_BIN': bin_backend_configmgr.name(),
             'SERVICE_ARGS': '--state-dir "' + openvpn3_statedir + '/configs"',
-            'OPENVPN_USERNAME': 'openvpn',
         }
     ),
     install: true,
diff --git a/src/log/meson.build b/src/log/meson.build
index 075c230..f2c00bd 100644
--- a/src/log/meson.build
+++ b/src/log/meson.build
@@ -44,7 +44,6 @@ configure_file(
             'BUSNAME': 'net.openvpn.v3.log',
             'SERVICE_BIN': bin_backend_log.name(),
             'SERVICE_ARGS': '--state-dir "' + openvpn3_statedir + '"',
-            'OPENVPN_USERNAME': 'openvpn',
         }
     ),
     install: true,
diff --git a/src/sessionmgr/meson.build b/src/sessionmgr/meson.build
index af765bc..042b062 100644
--- a/src/sessionmgr/meson.build
+++ b/src/sessionmgr/meson.build
@@ -46,7 +46,6 @@ configure_file(
             'BUSNAME': 'net.openvpn.v3.sessions',
             'SERVICE_BIN': bin_backend_sessionmgr.name(),
             'SERVICE_ARGS': '',
-            'OPENVPN_USERNAME': 'openvpn',
         }
     ),
     install: true,
-- 
2.49.0

